### PR TITLE
Adding more discussion of editorial practice to TC

### PR DESCRIPTION
**Pull request by hcayless**
This PR is an attempt to implement #1849. Comments welcome!

### CODE REVIEW

File: P5/Source/Guidelines/en/TC-CriticalApparatus.xml [hunk 1]
```diff
@@ -700,9 +717,8 @@ Barcelona, 1518, Carles Amorós. </bibl>
       </list></p>
     <p>Both the location-referenced and the double end-point methods may be used with either
         <term>in-line</term> or <term>external</term> apparatus, the former dispersed within the
-      base text, the latter held in some separate location, within or outside the document with the
-      base text. The parallel segmentation method does not use the concept of a base text and may
-      only be used for in-line apparatus.</p>
+      base text, the latter held in some separate location, within or outside the document containing the
+      base text. The parallel segmentation method may only be used for in-line apparatus.</p>
```
Comment (raffazizzi) [inline P5/Source/Guidelines/en/TC-CriticalApparatus.xml:721]: Yes, good idea changing this. The parallel segmentation can absolutely use the concept of a base text.

File: P5/Source/Guidelines/en/TC-CriticalApparatus.xml [hunk 2]
```diff
@@ -21,18 +21,36 @@ $Id$
...
+  
   <p>Information about variant readings (whether or not represented by a critical apparatus in the
     source text) may be recorded in a series of <term>apparatus entries</term>, each entry
     documenting one <term>variation</term>, or set of readings, in the text. Elements for the
     apparatus entry and readings, and for the documentation of the witnesses whose readings are
     included in the apparatus, are described in section <ptr target="#TCAPLL"/>. Special tags for
     fragmentary witnesses are described in section <ptr target="#TCAPMI"/>. The available methods
     for embedding the apparatus in the rest of the text, or for linking an external apparatus to the
-    base text, are described in section <ptr target="#TCAPLK"/>. Finally, several extra attributes
+    edition text, are described in section <ptr target="#TCAPLK"/>. Finally, several extra attributes
```
Comment (raffazizzi) [inline P5/Source/Guidelines/en/TC-CriticalApparatus.xml:32]: While saying "base text" here is too partial to one approach, I find "edition text" unclear. Maybe: "to the text being edited"? Though which one?

File: P5/Source/Guidelines/en/TC-CriticalApparatus.xml [hunk 3]
```diff
@@ -898,7 +914,7 @@ Barcelona, 1518, Carles Amorós. </bibl>
         point of the text are expressed as variants on one another. In this method, no two
         variations can overlap, although they may nest. The texts compared are divided into matching segments all
         synchronized with one another. This permits direct comparison of any span of text in any
-        witness with that in any other witness. It is also very easy with this method for an
+        witness with that in any other witness. It is also very easy (if the apparatus is positive) for an
```
Comment (raffazizzi) [inline P5/Source/Guidelines/en/TC-CriticalApparatus.xml:917]: When the apparatus is positive (that is... ), it is also very easy...

**Comment by hcayless** [comment]
I claim this is ready, barring other suggestions.

**Comment by lb42** [comment]
"What is a nature of the source(s)? Are there many witnesses..." --> "What source documents are being considered? Are there..."

**Comment by martinascholger** [comment]
Looks good to me. @raffazizzi could you have a look at it again?

**Pull request merged**